Command/CronRunCommand.php

```diff
@@ -35,7 +35,8 @@ protected function configure()
             ->setDescription('Runs any currently schedule cron jobs')
             ->addArgument('job', InputArgument::OPTIONAL, 'Run only this job (if enabled)')
             ->addOption('force', null, InputOption::VALUE_NONE, 'Force schedule the current job.')
-            ->addOption('schedule_now', null, InputOption::VALUE_NONE, 'Temporary set the job schedule to now.');
+            ->addOption('schedule_now', null, InputOption::VALUE_NONE, 'Temporary set the job schedule to now.')
```

Comment:
Please change the option's word separator with `-` instead of `_`.

Reply:
OK I did it but shouldn't `schedule_now` follow the same pattern?